commandapi-platforms/commandapi-bukkit/commandapi-bukkit-nms/commandapi-bukkit-1.15/src/main/java/dev/jorel/commandapi/nms/NMS_1_15.java

```diff
@@ -974,4 +976,17 @@ public <T> T getMinecraftServer() {
 		}
 	}
 
+	@Override
+	public void syncCommands() {
+		// For some reason, 1.15 is being stupid and has a private syncCommands method
+		// Reflection it is!
+		try {
+			Method syncCommands = Class.forName("org.bukkit.craftbukkit.v1_15_R1.CraftServer").getDeclaredMethod("syncCommands");
+			syncCommands.setAccessible(true);
+			syncCommands.invoke(Bukkit.getServer());
+		} catch (ClassNotFoundException | NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
```

Comment:
Did you know that `ReflectiveOperationException` covers all of these four exceptions?

Reply:
No, I did not! That is very useful to know!